bundler.d/assets.rb

```diff
@@ -1,7 +1,8 @@
 group :assets do
   gem 'jquery-ui-rails', '~> 6.0'
   gem 'patternfly-sass', '~> 3.59.4'
-  gem 'gettext_i18n_rails_js', '~> 1.3.1'
+  gem 'gettext_i18n_rails_js', '~> 1.4'
+  gem 'po_to_json', '~> 1.1.0'
```

Comment:
Any particular reason to pin to `~> 1.1.0` instead of `~> 1.1`?

Reply:
No, not really